modules/33acrossBidAdapter.js

```diff
@@ -640,6 +651,31 @@ function _createSync({ siteId = 'zzz000000000003zzz', gdprConsent = {}, uspConse
   return sync;
 }
 
+function getTopMostAccessibleFrame() {
+  let mostAccessibleWindow = utils.getWindowSelf();
+
+  try {
+    while (mostAccessibleWindow.parent !== mostAccessibleWindow &&
+      mostAccessibleWindow.parent.document) {
+      mostAccessibleWindow = mostAccessibleWindow.parent;
+    }
+  } catch (err) {
+    // Do not throw an exception if we can't access the topmost frame.
```

Comment:
I don't think so, because a `cross-domain embedded prebid.js` is considered a valid scenario, and this scenario doesn't impact  the addressability id calculation.